api/customers/customers.go

```diff
@@ -279,6 +283,42 @@ func (ch *CustomerHandler) UpdatePhoto(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintln(w, string(res))
 }
 
+func (ch *CustomerHandler) UpdatePassword(w http.ResponseWriter, r *http.Request) {
+	tokens := ch.jwt.GetToken(r)
+	err := tokens.Valid()
+	if err != nil {
+		helpers.HTTPError(w, http.StatusBadRequest, err.Error())
+		return
+	}
+
+	requestedBody, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		helpers.HTTPError(w, http.StatusBadRequest, constants.CannotReadRequest)
+		return
+	}
+
+	var newPass GetPasswordRequest
+	err = json.Unmarshal(requestedBody, &newPass)
+	if err != nil {
+		helpers.HTTPError(w, http.StatusBadRequest, "Invalid json type")
```

Comment:
maybe use constants as the error, if you use the constants on the error above

Reply:
done jocelyn